.swm/doc-templates/component-or-service-overview-pqkt4.sw.md

```diff
@@ -0,0 +1,28 @@
+---
+id: pqkt4
+name: Component or Service Overview
+file_version: 1.0.2
+app_version: 0.7.1-1
+---
+
+## Introduction
+This doc gives a high level overview of {{COMPONENT NAME}}. It is located under {{use `/path` to select the folder where the component / service is implemented}}.
+
+## Main features
+The main features of {{COMPONENT NAME}} are:
+
+## Interface
+{{How can this component/service be accessed?}}
+
+<br/>
+<!-- TEMPLATE-swimm-snippet-placeholder -->
+Select an example of accessing this component/service
+
+<br/>
+
+## Main contributors
+{{Mention the main contributors to this module}}
```

Comment:
How about directory structure? Add more sections here, this seems a bit empty...
Use ideas from Content Suggestions (see here https://github.com/swimmio/swimm/blob/develop/packages/shared/src/content-suggestions/consts.ts)

Reply:
Regarding the 'main contributors', not sure how useful it really is here... at the very least maybe mention @mentions

Reply:
I extended it a bit, but I am also refraining from overwhelming the user with too many sections